styles.css

```
@@ -8,24 +10,535 @@
-        etc.
+        4.3 Articles
+        4.4 Services
+        4.5 Footer
     5. Media Queries
 ////////////////////////*/
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
 
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --purple: #5722CD;
+    --light-purple: #F2F0FF;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --light-gray: #F3F6F9;
+    --gray: #7A7A7A;
+    --white: #FFFFFF;
+}
+
+body {
+    font-family: 'Roboto', 'Arial', 'Helvetia', sans-serif;
+    font-weight: 300;
+    font-size: 20px;
+    color: var(--black);
+}
+
+
 /* ------------------------------- 2. Typography ----------------------------------- */
 
+h1, h2, h4, h5, h6 {
+    font-family: 'Merriweather', 'Times New Roman', serif;
+    color: var(--white);
+    font-size: 41px;
+    font-weight: 700;
+}
+
+.header-text-succeed {
+    font-weight: 300;
+    font-size: 25px;
+    line-height: 85px;
+}
+
+
+h4 {
+    color: var(--purple);
+    font-size: 29px;
+    font-weight: 400;
+}
+
+.article__text-container h4 {
+    padding: 7px 0;
+    margin: 87px 0 0 0;
+    color: var(--black);
+}
+
+p {
+    font-family: 'Roboto', 'Arial', sans-serif;
+    color: var(--black);
+    font-size: 18px;
+    font-weight: 400;
+}
+
+
+.header-navigation a {
+    text-decoration: none;
+    color: white;
+}
+
 /* ------------------------------- 3. Layout --------------------------------------- */
 
 /* --------------- 3.1 Reusables ----------------------- */
 
+.outer-container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.inner-container {
+    max-width: 767px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 20px;
+    flex-shrink: 1;
+    flex-grow: 0;
+
+}
+
+
 /* --------------- 3.2 Other --------------------------- */
 
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
 
+.header-section-container {
+    position: relative;
+    background: rgb(72, 69, 228);
+    background: linear-gradient(133deg, rgba(72, 69, 228, 1) 26%, rgba(87, 34, 205, 1) 100%);
+    color: var(--white);
+    justify-content: space-between;
+    padding-bottom: 160px;
+
+}
+
+.header-navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+    padding: 29px 53px;
+}
+
+.header-navigation ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 361px;
+    list-style: none;
+    flex-shrink: 1;
+    flex-grow: 0;
+
+}
+
+.header-navigation li {
+    flex-basis: auto;
+    height: 30px;
+    flex-shrink: 1;
+}
+
+.header-text-container {
+    margin: 65px;
+    text-align: center;
+}
+
+button {
+    background-color: var(--pink);
+    padding: 23px 39px;
+    border-radius: 50px 50px 50px 50px;
```

Dit kun je verkorten:
```suggestion
    border-radius: 50px;
```